### PR TITLE
redux-localstorage, dehydrated detector logout() fix

### PR DESCRIPTION
**Pull request by mattmcfad**
Connected to rangle/rangle-starter#17

### CODE REVIEW

File: src/containers/app.ts [hunk 1]
```diff
@@ -51,8 +51,8 @@ import {
               session.getIn(['user', 'firstName'], '') + ' ' +
               session.getIn(['user', 'lastName'], '') }}
           </navigator-item>
-          <navigator-item *ngIf="isLoggedIn">
-            <btn (click)="logout()" class="bg-red white">
```
Comment (mattmcfad) [inline src/containers/app.ts]: This was causing an `Error: Attempt to use a dehydrated detector.` which I haven't been able to find a solution for using `*ngIf`. When logout is triggered the `*ngIf` will remove the `btn` from the DOM, which the button's `ChangeDetector` doesn't like.

The work around is using `[hidden]` or `[ngClass]`, but this is not consistent with how the rest of the app is being shown/hidden and also keeps the button in the DOM.

@rpdasilva @alanthai

Comment (mattmcfad) [inline src/containers/app.ts]: This was causing an `Error: Attempt to use a dehydrated detector.` which I haven't been able to find a solution for using `*ngIf`. When logout is triggered the `*ngIf` will remove the `btn` from the DOM, which the button's `ChangeDetector` doesn't like.

The work around is using `[hidden]` or `[ngClass]`, but this is not consistent with how the rest of the app is being shown/hidden and also keeps the button in the DOM.

@rpdasilva @alanthai

File: src/containers/app.ts [hunk 2]
```diff
@@ -51,8 +51,8 @@ import {
...
+          <navigator-item [hidden]="!isLoggedIn">
```
Comment (SethDavenport) [inline src/containers/app.ts]: thought we were going with ngIf?

Comment (SethDavenport) [inline src/containers/app.ts]: OK that's fine.

**Comment by SethDavenport** [comment]
:shipit:

**Pull request merged**